tests/test_search.py

```diff
@@ -461,15 +461,16 @@ def test_search_and_parse(self, shortname, string, expected, settings=None):
                     January UTC
                     June 5 am utc
                     June 23th 5 pm EST
-                    May 31, 8am UTC""",
+                    May 31
+                    8am UTC""",
```

Comment:
Why this change in the test input?

Reply:
These changes just look clear the output is the same. Should I change it to how it was.

From
```
May 31, 8am UTC
```

To 

```
May 31
8am UTC
```

Reply:
Yes, I think that would be best. If we stopped supporting that comma in there, this change would prevent the test from failing accordingly. Also, I think `May 31, 8am UTC` is more readable, as that is meant to be parsed as a single date (regardless of whether or not `search_dates` does so at the moment).

Reply:
Hi @gavishpoddar, could you revert this as @Gallaecio requested? :)

Reply:
I have made the changes

Reply:
As far as I can tell you have not, the change still appears in the pull request.

Reply:
I’m thinking that I may not have been clear. The line used to have a comma, you changed it to a line break, and I’m saying it should be a comma again.

Reply:
Thanks, @Gallaecio, now I understood it.